docker/testrunner/run.sh

```diff
@@ -0,0 +1,113 @@
+#!/bin/bash
```

Comment:
Can we make it dependent on `build.sh`? Meaning that if I execute `run.sh` and there is no container yet, it would build it.